.pre-commit-config.yaml

```diff
@@ -17,19 +17,19 @@ repos:
       - id: check-added-large-files
 
   - repo: https://github.com/psf/black
-    rev: 3702ba224ecffbcec30af640c149f231d90aebdb # frozen: 24.4.2
+    rev: 1b2427a2b785cc4aac97c19bb4b9a0de063f9547 # frozen: 24.10.0
     hooks:
       - id: black
 
   - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
This repo has the notice on it: This repository has been archived by the owner on Apr 11, 2024. It is now read-only.

Are we still supposed to use it (the mirror), or is there a way to use prettier directly?

For now, I would ignore this version update.